kernel-api/src/main/scala/org/apache/toree/interpreter/broker/BrokerTransformer.scala

```diff
@@ -43,7 +43,7 @@ class BrokerTransformer {
     import scala.concurrent.ExecutionContext.Implicits.global
 
     futureResult
-      .map(results => (Results.Success, Left(results)))
+      .map(results => (Results.Success, Left(Map("text/plain" -> results))))
```

Comment:
No, kernel-api doesn't depend on protocol so it isn't available. But I think that this code shouldn't be sending mime types in the first place. Sending text/plain here happens because we haven't updated the Python kernel to correctly inspect its objects.

Eventually, the python interpreter should send back a result that is a py4j reference to the final value and the python representation (from _repr_ and _repr_html_) or a JVM object and None to indicate that the Jupyter JVM displayer should be called. So for now, I'd rather not change module dependencies.